plugins/helphub-post-types/classes/class-helphub-post-types-post-type.php

```diff
@@ -393,11 +404,11 @@ public function meta_box_content_render( $field_data ) {
 						$html .= '<tr valign="top"><th scope="row"><label for="' . esc_attr( $v['name'] ) . '">' . $v['name'] . '</label></th><td>' . $field . "\n";
 						$html .= '</td></tr>' . "\n";
 						break;
-					case 'multicheck':
+					case 'multicheck' :
```

Comment:
Case statements should not have a space before the colon, i.e. `'case 'multicheck':`